packages/relay/src/formatters.ts

```diff
@@ -125,12 +125,16 @@ const parseNumericEnvVar = (envVarName: string, fallbackConstantKey: string): nu
  * @returns tinybarValue
  */
 const weibarHexToTinyBarInt = (value: string): number | null => {
+    if ((value === null) || (value === '0x')) {
```

Comment:
Simpler to just add `value !== '0x'` to the if statement below

Reply:
Done.

Reply:
Suggested an improvement